frontend/components/JobApplicationsDashboard.tsx

```diff
@@ -342,32 +342,42 @@ export default function JobApplicationsDashboard({
 				<div className="overflow-x-auto bg-white dark:bg-black shadow-md rounded-lg">
 					<Table aria-label="Applications Table">
 						<TableHeader>
-							<TableColumn>Company</TableColumn>
-							<TableColumn>Status</TableColumn>
-							<TableColumn>Received</TableColumn>
-							<TableColumn>Job Title</TableColumn>
-							<TableColumn>Subject</TableColumn>
-							<TableColumn>Sender</TableColumn>
-							<TableColumn>Actions</TableColumn>
+							<TableColumn className="text-center">Company</TableColumn>
+							<TableColumn className="text-center">Status</TableColumn>
+							<TableColumn className="text-center">Received</TableColumn>
+							<TableColumn className="text-center">Job Title</TableColumn>
+							<TableColumn className="text-center">Subject</TableColumn>
+							<TableColumn className="text-center">Sender</TableColumn>
+							<TableColumn className="text-center">Actions</TableColumn>
 						</TableHeader>
 						<TableBody>
 							{paginatedData.map((item) => (
 								<TableRow
 									key={item.id || item.received_at}
 									className="hover:bg-default-100 transition-colors"
 								>
-									<TableCell>{item.company_name || "--"}</TableCell>
-									<TableCell>
+									<TableCell className="max-w-[100px] text-center">
+										{item.company_name || "--"}
+									</TableCell>
+									<TableCell className="max-w-[120px] break-words whitespace-normal text-center">
 										<span
-											className={`inline-flex items-center justify-center px-2 py-1 rounded text-sm font-medium ${getStatusClass(item.application_status)}`}
+											className={`inline-flex items-center justify-center px-1.5 py-1 rounded text-s font-medium ${getStatusClass(item.application_status)}`}
```

Comment:
```suggestion
											className={`inline-flex items-center justify-center px-1.5 py-1 rounded text-sm font-medium ${getStatusClass(item.application_status)}`}
```

Text-s doesn't exist in tailwind class I don't think 🤔